KubeArmor/eventAuditor/entryPointHandler.go

```diff
@@ -132,30 +133,44 @@ func (ea *EventAuditor) DestroyEntryPointPrograms(bman *KABPFManager) error {
 func (ea *EventAuditor) EnableEntryPoint(probe string) {
 	var eventMapElem EventElement
 
+	probe = strings.Replace(probe, "sys_", "", -1)
+	_, supported := ea.SupportedEntryPoints[probe]
+
+	if !supported {
+		fmt.Println(probe, "is currently not supported")
+		return
+	}
+
 	eventMapElem.SetKey(uint32(ea.SupportedEntryPoints[probe]))
 	eventMapElem.SetValue(uint32(1))
 
-	// TODO: value is not updated
 	if err := ea.BPFManager.MapUpdateElement(&eventMapElem); err != nil {
 		ea.Logger.Errf("Failed to update KAEAEventMap (attachEntryPoint, %s, %d) (%v)", probe, err)
 	}
 
-	// TODO: remove it once fixing the update
+	// for debugging
```

Comment:
If the logic works, we can remove 151-152.

Reply:
We are not getting the required output from the trace_pipe, even when the map is being correctly updated. This part was left for debugging this error.

Reply:
you mean the current code is not working?

Reply:
yes, there is some issue with the "val" in entrypoint.bpf.c, we are not getting the required output from the trace_pipe.
The syscalls whose value is set to 1 (and are enabled), are being skipped.